wordpress-mobile/WordPress-Android
# Contact support launch chat from all entry points

Launch chat from the following places as well
- Log-in
- Editor


Fixes #18992 

To test:

Setup
- Enable feature flag (Me -> Debug settings -> `contact_support_chatbot`)

Log in
- Launch JP app
- Choose Log in through Enter existing site address
- On the next screen `Tap` on `Find my site address`
- On the pop-up `Tap` on `Need more help?`
- `Verify` it launches Chat, and chat works as  expected

Editor
- Create a post (FAB -> Blog post)
- in the Editor `Tap` on `three dot menu` on top right hand corner next to PUBLISH
- Select `Help & Support`
- `Tap` on `Contact support` below Get support
- `Verify` it launches Chat

>**Note**
> Feature flag name is updated to `contact_support_chatbot` to match the remote flag

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual tests

3. What automated tests I added (or what prevented me from doing so)
Existing unit tests

PR submission checklist:
